<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="_Tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="SimulationWindow" name="simulationWindow">
     <attribute name="title">
      <string>Symulacja</string>
     </attribute>
    </widget>
    <widget class="SceneWindow" name="sceneWindow">
     <attribute name="title">
      <string>Scena</string>
     </attribute>
    </widget>
    <widget class="AgentWindow" name="agentWindow">
     <attribute name="title">
      <string>Agent</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>579</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGjh">
   <property name="text">
    <string>gjh</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SimulationWindow</class>
   <extends>QWidget</extends>
   <header location="global">simulationwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AgentWindow</class>
   <extends>QWidget</extends>
   <header location="global">agentwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SceneWindow</class>
   <extends>QWidget</extends>
   <header location="global">scenewindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
